src/jshal.js

```diff
@@ -26,19 +26,15 @@
 
 mergeInto(LibraryManager.library, {
   mp_js_hal_init: async function () {
-    MP_JS_EPOCH = new Date().getTime();
-    board.initialize({
-      defaultAudioCallback: window.microbit_hal_audio_ready_callback,
-      speechAudioCallback: window.microbit_hal_audio_speech_ready_callback,
-    });
+    board.initialize();
   },
```

Comment:
This was just an unused param. Would be really good to get type checking working in this file.